templates/default/compute_ready.erb

```diff
@@ -10,4 +10,4 @@ instance_type_url="http://169.254.169.254/latest/meta-data/instance-type"
 instance_type=$(curl --retry 3 --retry-delay 0 --silent --fail ${instance_type_url})
 local_hostname_url="http://169.254.169.254/latest/meta-data/local-hostname"
 local_hostname=$(curl --retry 3 --retry-delay 0 --silent --fail ${local_hostname_url})
-<%= node['cfncluster']['cookbook_virtualenv_path'] %>/bin/aws --region ${cfn_region} sqs send-message --queue-url ${cfn_sqs_queue} --message-body '{"Type" : "Notification", "Message" : "{\"StatusCode\":\"Complete\",\"Description\":\"Succesfully launched '${instance_id}'\",\"Event\":\"parallelcluster:COMPUTE_READY\",\"EC2InstanceId\":\"'${instance_id}'\",\"Slots\":\"'${cfn_instance_slots}'\",\"LocalHostname\":\"'${local_hostname}'\",\"EC2InstanceType\":\"'${instance_type}'\"}"}'
+<%= node['cfncluster']['cookbook_virtualenv_path'] %>/bin/aws --region ${cfn_region} sqs send-message --queue-url ${cfn_sqs_queue} --endpoint https://sqs.${cfn_region}.amazonaws.com$([[ ${cfn_region} =~ ^cn- ]] && echo ".cn") --message-body '{"Type" : "Notification", "Message" : "{\"StatusCode\":\"Complete\",\"Description\":\"Succesfully launched '${instance_id}'\",\"Event\":\"parallelcluster:COMPUTE_READY\",\"EC2InstanceId\":\"'${instance_id}'\",\"Slots\":\"'${cfn_instance_slots}'\",\"LocalHostname\":\"'${local_hostname}'\",\"EC2InstanceType\":\"'${instance_type}'\"}"}'
```

Comment:
are we sure this is not `--endpoint-url`?

Reply:
Thanks for your checking.
Yes, you are definitely right. `--endpoint-url` is correct.
In my excuse, I found (and surprised) both `--endpoint` and `--endpoint-url` could work.
Anyway, I fixed it.